Showcase/Showcase.md

```diff
@@ -100,7 +102,6 @@ Without aligned columns
 
 | Name                 | CommandType | Version |
 | -------------------- | ----------- | ------- |
-| New-MDAlert          | Function    | 0.0     |
```

Comment:
Why was removed?

Reply:
I am not sure on this one, I did not manually edit showcase.md, i just ran the Update-Showcase.ps1 script. let me check how it changed.

Reply:
Let me know. Thanks.